src/esp/gfx/RenderTarget.cpp

```diff
@@ -114,6 +114,17 @@ struct RenderTarget::Impl {
 
   void renderExit() {}
 
+  void blitRgbaToDefault() {
+    GL::defaultFramebuffer.clearColor(Color4{1, 0, 0, 1});
+    GL::defaultFramebuffer
+        .clear(GL::FramebufferClear::Color | GL::FramebufferClear::Depth)
+        .bind();
```

Comment:
I don't think the bind is needed. blit will bind.

Reply:
Yeah, the call to `GL::AbstractFramebuffer::blit` should be the only thing you need to do here.